### ISSUE TITLE
coefstat broken in fixest_0.8.4 but works in 0.8.2

### ISSUE THREAD

**Issue by EdJeeOnGitHub** [lrberge/fixest#132]
Hi,

Thanks so much for all  your work on `fixest`!

`etable()`'s `coefstat` option will only support `"se"` and return an error for `"tstat", "confint"` in `fixest_0.8.4` but works fine for `_0.8.2` - I haven't tested 0.8.3.

``` r
library(fixest)
fit <- feols(speed ~ dist, cars)
etable(fit, coefstat = "confint")
#> Error: in etable(fit, coefstat = "confint"):
#>  Argument 'coefstat' must be a single character equal to 'se'. Problem:
#> no match was found for 'confint'.
etable(fit, coefstat = "tstat")
#> Error: in etable(fit, coefstat = "tstat"):
#>  Argument 'coefstat' must be a single character equal to 'se'. Problem:
#> no match was found for 'tstat'.
sessionInfo()
#> R version 4.0.5 (2021-03-31)
#> Platform: x86_64-pc-linux-gnu (64-bit)
#> Running under: Ubuntu 20.04 LTS
#> 
#> Matrix products: default
#> BLAS/LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.8.so
#> 
#> locale:
#>  [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
#>  [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
#>  [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=C             
#>  [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
#>  [9] LC_ADDRESS=C               LC_TELEPHONE=C            
#> [11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       
#> 
#> attached base packages:
#> [1] stats     graphics  grDevices utils     datasets  methods   base     
#> 
#> other attached packages:
#> [1] fixest_0.8.4
#> 
#> loaded via a namespace (and not attached):
#>  [1] Rcpp_1.0.6          Formula_1.2-4       knitr_1.31         
#>  [4] magrittr_2.0.1      lattice_0.20-41     rlang_0.4.10       
#>  [7] fansi_0.4.2         stringr_1.4.0       styler_1.4.1       
#> [10] highr_0.8           tools_4.0.5         grid_4.0.5         
#> [13] nlme_3.1-152        xfun_0.22           utf8_1.2.1         
#> [16] withr_2.4.1         htmltools_0.5.1.1   ellipsis_0.3.1     
#> [19] yaml_2.2.1          digest_0.6.27       tibble_3.1.0       
#> [22] lifecycle_1.0.0     numDeriv_2016.8-1.1 crayon_1.4.1       
#> [25] purrr_0.3.4         vctrs_0.3.7         fs_1.5.0           
#> [28] glue_1.4.2          evaluate_0.14       rmarkdown_2.7      
#> [31] sandwich_3.0-0      reprex_2.0.0        stringi_1.5.3      
#> [34] dreamerr_1.2.3      compiler_4.0.5      pillar_1.5.1       
#> [37] backports_1.2.1     zoo_1.8-9           pkgconfig_2.0.3
```

<sup>Created on 2021-04-21 by the [reprex package](https://reprex.tidyverse.org) (v2.0.0)</sup>

**Comment by EdJeeOnGitHub** [comment]
But it's fixed in 0.9.0!

My bad - thanks again.

**Comment by lrberge** [comment]
Great, thanks for finding this out! :-) and thanks for the words!
